docs/root/configuration/http/http_filters/bandwidth_limit_filter.rst

```diff
@@ -0,0 +1,61 @@
+.. _config_http_filters_bandwidth_limit:
+
+HTTP Bandwidth limit
+====================
+
+* Bandwidth limiting :ref:`architecture overview <arch_overview_bandwidth_limit>`
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.bandwidth_limit.v3alpha.BandwidthLimit>`
+* This filter should be configured with the name ``envoy.filters.http.bandwidth_limit``.
+
+The HTTP Bandwidth limit filter limits the size of data flow to the max bandwidth set in the limit_kbps
+when the request's route or virtual host has a per filter
+:ref:`bandwidth limit configuration <envoy_v3_api_msg_extensions.filters.http.bandwidth_limit.v3alpha.BandwidthLimit>`.
+
+If the bandwidth limit has been exhausted the filter stops further transfer until more bandwidth gets allocated
+according to the ``fill_interval`` (default is 50 milliseconds). If the connection buffer fills up with accumulated
+data then the source of data will be readDisabled as described in the :ref:`flow control doc<faq_flow_control>`.
```

Comment:
clicking through to the flow control doc i dont see any further info on what `readDisabled` means

Reply:
thanks. Actually, I intended to link to this doc: https://github.com/envoyproxy/envoy/blob/main/source/docs/flow_control.md. Fixed.

Reply:
could you make `readDisabled` literal in that case  - also use `:repo:` directive for linking repo (if you havent done so already)

Reply:
Fixed the link but didn't make 'readDisabled' literal since it is not code.